tiledb/api/src/array/attribute/mod.rs

```diff
@@ -345,6 +365,25 @@ impl Builder {
         Ok(self)
     }
 
+    /// Set the name of the enumeration to use.
+    ///
+    /// Note that when building schemas, the enumeration must have been added
+    /// to the schema for adding an attribute that references it.
```

Comment:
Does failing to abide by this result in an error here?  Or when the attribute is added to the schema? Or at schema creation time?

Reply:
You'll get an error from TileDB core yeah: https://github.com/TileDB-Inc/TileDB/blob/dev/tiledb/sm/array_schema/array_schema.cc#L810-L820